cookbooks/aws-parallelcluster-install/attributes/default_ubuntu20.rb

```diff
@@ -1,9 +0,0 @@
-# Ubuntu 20 default attributes for aws-parallelcluster-install
-
-return unless platform?('ubuntu') && node['platform_version'] == "20.04"
-
-default['nfs']['service_provider']['idmap'] = Chef::Provider::Service::Systemd
```

Comment:
If I'm not wrong these attributes are defined here to override properties of the nfs third party cookbook.
Are we sure that is safe to remove them?

Reply:
I hope our tests (see description) are sufficient to certify we are safe to remove these attributes. If not, we will integrate them with missing checks.